### PR TITLE
Tcsp2

### PR DESCRIPTION
**Pull request by finkeljos**
Added tensor core functionality for density matrix construction.

---
This change is [Reviewable](https://reviewable.io/reviews/lanl/qmd-progress/229)

### CODE REVIEW

**Review by cnegre — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -125,6 +125,12 @@ else()
     add_definitions(-DMETIS_REAL_KIND=0d0)
 endif()
 
+if(NOT PROGRESS_SP2TC MATCHES "no")
```
Comment (nicolasbock) [inline CMakeLists.txt:128]: This is matching on a regular expression. Why not simply check for equality?

**Comment by cnegre** [comment]
Closing due to merging conflicts

**Review by nicolasbock — COMMENTED** [review]
Why did you close this PR @cnegre ?

**Comment by nicolasbock** [comment]
The branch has a few merges in it itself which makes it difficult to rebase onto `master`.

**Comment by cnegre** [comment]
I could not find an easy way to merge the branch into master. We will send
another pr with changes on top of master.

**Pull request closed without merging**